### ISSUE TITLE
Login Page Animation

### ISSUE THREAD

**Issue by geektype** [omanomanom/site#1]
# Login page Animation
@omanomanom added base switch
Add annimation to the login-registeration toggle

**Comment by geektype** [comment]
How about we just get rid of the registeration and ake it seperate. Check [this](https://codepen.io/jdniki/pen/rewxPo) out. So have a step by step registration. It just looks good

**Comment by omanomanom** [comment]
> How about we just get rid of the registeration and ake it seperate. Check [this](https://codepen.io/jdniki/pen/rewxPo) out. So have a step by step registration. It just looks good

no, create animation, looks and feels better which is how i want site to work
